src/main/java/core/basesyntax/Writer.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+
+public class Writer {
+    public void createCsvFile(String toFileName) {
+        File file = new File(toFileName);
+        try {
+            file.createNewFile();
+        } catch (IOException e) {
+            throw new RuntimeException("Can't create file " + toFileName, e);
+        }
+    }
```

Comment:
The method name 'createCsvFile' should be in lowerCamelCase according to Java naming conventions.